css/index.css

```diff
@@ -1,3 +1,410 @@
+                                                    body {
+                                                        width: 100%;
+                                                        margin: 0 auto;
+                                                        margin-right: 550px;
+                                                    }
+                                                    .container {
+                                                        width: 100%;
+                                                    }
+                                                    nav {
+                                                        flex-wrap: wrap;
+                                                        justify-content: space-around;
+                                                        margin-left: 10px;
+                                                    }
+                                                    nav a {
+                                                        padding: 15px;
+                                                    }
+                                                    nav a:last child {
+                                                        order: -1;
+                                                        width: 100%;
+                                                    }
+                                                    .imm {
+                                                        display: none;
+                                                    }
+                                                    .middle-img {
+                                                        width: 100%;
+                                                    }
+                                                    .two-headers {
+                                                        width: 100%;
+                                                    }
+                                                    .three-header {
+                                                        width: 100%;
+                                                    }
+                                                    footer {
+                                                        width: 100%;
+                                                    }
+                                                    .anchor {
+                                                        order: 1;
+                                                    }
+                                                    .a-logo {
+                                                        order: 0;
+                                                        width: 100%;
+                                                        text-align: center;
+                                                    }
+                                                    /* mobile */
+                                                    @media only screen and (max-width: 500px) {
```

Comment:
You're still getting horizontal scroll bars on your breakpoints. Try making your container smaller than 100% of the screensize and see if that fixes things.